packages/react-native/local-cli/runMacOS/runMacOS.js

```diff
@@ -47,6 +48,13 @@ function runMacOS(_, _ctx, args) {
     );
   }
 
+  if (args.configuration) {
+    logger.warn(
+      'Argument --configuration has been deprecated and will be removed in a future release, please use --mode instead.',
+    );
+    args.mode = args.configuration;
```

Comment:
If `mode` is already set, wouldn't this override it?

Reply:
Yeah, true. Fixed.